examples/0_Simple/simpleStreams/src/host/main.adb

```diff
@@ -74,7 +74,11 @@ begin
    Set_Device_Flags (Device_Sync_Method or (if B_Pin_Generic_Memory then cudaDeviceMapHost else 0));
 
    Host_A := new Integer_Array'(0 .. N - 1 => 0);
-   Device_A := new Integer_Array'(0 .. N - 1 => 0);
+
+   --Device_A := new Integer_Array'(0 .. N - 1 => 0);
```

Comment:
I suggest adding an explanation of why that line is commented out.

Reply:
Definitely - at least up until this is fixed.